apps/edxapp/vars/all/main.yml

```diff
@@ -87,6 +87,7 @@ edxapp_mongodb_host: "edxapp-mongodb"
 edxapp_mongodb_replicaset: ""
 edxapp_mongodb_read_preference: ""
 edxapp_mongodb_secret_name: "edxapp-mongo-{{ edxapp_vault_checksum | default('undefined_edxapp_vault_checksum') }}"
+edxapp_endpoint_mongodb_ips: "{{ endpoint_mongodb_ips }}"
```

Comment:
```suggestion
edxapp_endpoint_mongodb_ips: {{ endpoint_mongodb_ips }}
```

Reply:
No it doesn't work, I test it

Reply:
If this entry is casted as a list despite quotes forget it then!